Resources/Locale/ru-RU/_CP14/_PROTO/entities/objects.ftl

```diff
@@ -18,6 +18,8 @@ ent-CP14BaseThrowableSpear = метательное копьё
     .desc = Оружие, исправно выполняющее свой долг еще со времен эпохи великанов.
 ent-CP14BaseTwoHandedSword = двуручный меч
     .desc = Мощное оружие, требующее огромной силы и умения для эффективного использования.
+ent-CP14OldLantern = Старая Лампа
```

Comment:
put it not in weapon category. Its sorted as prototype folders, so it need 
# Tools

Reply:
Ups, my bad